tests/behat/features/bootstrap/SocialDrupalContext.php

```diff
@@ -496,4 +496,36 @@ public function createNodeLikes(TableNode $nodesTable): void {
     }
   }
 
+  /**
+   * I wait for (field_name) (field_type) be rendered.
+   *
+   * @When /^(?:|I )wait for "([^"]*)" "([^"]*)" field be rendered$/
```

Comment:
Tip for next time is to avoid using regex here but use the "simple form" that Behat supports:

```php
    * @When (I )wait for the :field_name :field_type field to be rendered
```

It's slightly easier to read because it doesn't require knowledge of Regex. It also has the advantage of not requiring the double quotes (`"`) in your `.feature` file step. So the two usages from this PR:

```
And I wait for "Enroll as guest" "link" field be rendered
And I wait for "field_first_name" "input" field be rendered
```

Can become

```
And I wait for "Enroll as guest" link field be rendered
And I wait for field_first_name input field be rendered
```

PHPStorm should then still be smart enough to highlight "Enroll as guest", link, field_first_name, and input as being changeable placeholders :D